### PR TITLE
[Kotlin Migration] DatabaseUtils

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] DatabaseUtils
Package: com.ichi2.anki.testutil

### CODE REVIEW

**Review by david-allison — CHANGES_REQUESTED** [review]
Could you rename `position_`: 

probably changing `int position` to `positionParam` and `position_` to `position`

**Comment by codingtosh** [comment]
> Could you rename `position_`:
> 
> probably changing `int position` to `positionParam` and `position_` to `position`

@david-allison Done, PTAL.

File: AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/DatabaseUtils.kt [hunk 1]
```diff
@@ -49,60 +48,54 @@ public class DatabaseUtils {
      * @param position The start position for filling the window.
```
Comment (david-allison) [inline AnkiDroid/src/androidTest/java/com/ichi2/anki/testutil/DatabaseUtils.kt:48]: no longer matches

**Review by david-allison — CHANGES_REQUESTED** [review]

**Review by mikehardy — APPROVED** [review]
LGTM?

**Review by david-allison — APPROVED** [review]

**Pull request merged**